packages/picolisp/build.sh

```diff
@@ -1,84 +1,77 @@
 TERMUX_PKG_HOMEPAGE=https://picolisp.com/wiki/?home
 TERMUX_PKG_DESCRIPTION="Lisp interpreter and application server framework"
 TERMUX_PKG_LICENSE="MIT"
-# TERMUX_PKG_SRCDIR is overriden below
-TERMUX_PKG_LICENSE_FILE="../COPYING"
+TERMUX_PKG_LICENSE_FILE="COPYING"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=20.7.24
-TERMUX_PKG_REVISION=2
-# We use our source copies since old version snapshots are not kept on main site.
-TERMUX_PKG_SRCURL=https://github.com/termux/distfiles/releases/download/2021.01.04/picolisp_${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=72e8d394ad32a6659210ac2cc4b7bb736dc9b94c6ac8d6296506b6dfdc92f90c
-TERMUX_PKG_DEPENDS="libcrypt, openssl"
+TERMUX_PKG_VERSION=21.12
+TERMUX_PKG_SRCURL=http://archive.ubuntu.com/ubuntu/pool/universe/p/picolisp/picolisp_$TERMUX_PKG_VERSION.orig.tar.gz
+TERMUX_PKG_SHA256=a06838236b7f5b52c5d587d32d31627f73cdb9775cc02a80f2cdaedd12888c7d
+# Clang is required for compiling sysdefs.c and running the binary which is not possible while cross compiling
```

Comment:
Depending on `clang` solely for that purpose feels somewhat overkill. How about cross-compiling `sysdefs.c` and include the binary that is to be run in `postinst`?

Reply:
Fixed, thanks!